src/openms/source/ANALYSIS/ID/AhoCorasickAmbiguous.cpp

```diff
@@ -278,8 +278,9 @@ namespace OpenMS
       {
         const auto needle_length = trie_[i()].depth_and_hits.depth;
         const auto text_start = text_pos - needle_length;
-        for (const auto needle_idx : umap_index2needles_.at(i())) {
-          hits.emplace_back(needle_idx, needle_length, text_start);
+        for (const auto needle_idx : umap_index2needles_.at(i()))
+        {
+          hits.emplace_back(needle_idx, needle_length, Hit::T(text_start));
```

Comment:
constructor? conversion?

Reply:
conversion at the moment.
I wanted to force users to make a conscious decision about it.